shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -505,6 +557,9 @@ @implementation FlutterTextInputView {
   const char* _selectionAffinity;
   FlutterTextRange* _selectedTextRange;
   CGRect _cachedFirstRect;
+  NSArray* _selectionRects;
```

Comment:
Use property instead of ivar so the memory management is explicit.  I don't see a release for this array so it's probably leaking.